other/bootstrap_daemon/README.md

```diff
@@ -1,27 +1,25 @@
-##Instructions for Debian
+##Instructions 
 
 For security reasons we run the daemon under its own user.
 Create a new user by executing the following:
 ```sh
-sudo useradd --system --shell /sbin/nologin --comment "Account to run Tox's DHT bootstrap daemon" --user-group tox-bootstrapd
+sudo useradd --home-dir /var/lib/tox-bootstrapd --create-home --system --shell /sbin/nologin --comment "Account to run Tox's DHT bootstrap daemon" --user-group tox-bootstrapd
 ```
 
-Create a directory where the daemon will store its keys:
-```sh
-sudo mkdir /var/lib/tox-bootstrapd/
-```
+Place `tox-bootstrapd.conf` file to where `CFGFILE` variable from `tox-bootstrapd.sh` tells (for `init.d` users) or `ExecStart=` from `tox_bootstrap.service` ( for `systemd` users). By default it's `/etc/tox-bootstrapd.conf`.
+
+Go over everything in `tox-bootstrapd.conf`. Make sure `pid_file_path` matches `PIDFILE` from `tox-bootstrapd.sh` (`init.d`) or `PIDFile=` from `tox_bootstrap.service` AND file in `ExecStartPre`(`systemd`).
+
 
-Restrain other users from accessing the directory:
+Set privileges and ownership:
 ```sh
-sudo chown tox-bootstrapd:tox-bootstrapd /var/lib/tox-bootstrapd/
-sudo chmod 700 /var/lib/tox-bootstrapd/
+chown tox-bootstrapd:tox-bootstrapd -R /var/lib/tox-bootstrapd
```

Comment:
Isn't this redundant because of the permissions set when creating the user earlier?